controllers/storageconsumer/storageconsumer_upgrade_controller.go

```diff
@@ -219,6 +220,13 @@ func (r *StorageConsumerUpgradeReconciler) reconcileStorageRequest(
 	rbdStorageRequest.SetGroupVersionKind(ocsv1alpha1.GroupVersion.WithKind("StorageRequest"))
 	rbdStorageRequest.Name = rbdStorageRequestName
 	rbdStorageRequest.Namespace = storageConsumer.Namespace
+	if err := r.Client.Get(ctx, client.ObjectKeyFromObject(rbdStorageRequest), rbdStorageRequest); client.IgnoreNotFound(err) != nil {
+		return err
+	}
+	rbdStorageRequest.Finalizers = []string{}
+	if err := r.Client.Patch(ctx, rbdStorageRequest, client.MergeFrom(rbdStorageRequest)); client.IgnoreNotFound(err) != nil {
```

Comment:
There is no reason to resort to patch here. Can we please use the more common and understood update operation.

Reply:
Ok, resolving